class-components/src/components/BottomSection/BottomSection.type.ts

```diff
@@ -0,0 +1,75 @@
+import { apiLink } from '../../api/api';
+export interface SearchState {
+  searchTerm: string;
+}
+
+export interface ApiResponse<T> {
+  count: number;
+  next: string | null;
+  previous: string | null;
+  results: T[];
+}
+
+export interface ApiError {
+  detail: string;
+}
```

Comment:
I didn't find any instances of usage of `ApiResponse` and `ApiError  `in the project. Do you need them? If you do, please apply the interfaces in the relevant places.

Also, please move them to a separate file in `api `folder as the types should be placed where they are first and most logically used